git-hooks/pre-commit

```diff
@@ -7,16 +7,16 @@
 
 # git gofmt pre-commit hook
 #
-# To use, store as .git/hooks/pre-commit inside your repository and make sure
-# it has execute permissions.
+# To use, execute script/install-git-hooks and all the hooks will
+# by installed via symlink.
 #
 # This script does not handle file names that contain spaces.
 
 gofiles=$(git diff --cached --name-only --diff-filter=ACM | grep '.go$')
-[ -z "$gofiles" ] && exit 0
+[[ -z "$gofiles" ]] && exit 0
```

Comment:
these were OK with the single-`[`s I think. Would rather not reflexively change it unless you have some practical reason I'm not seein'

Reply:
This is generally safer since no wordsplitting or glob expansion will be done for [[..]] versus using [..]  If you run shellcheck linter against this, it will not pass the linter.  I tend to use shellcheck to lint all of my shell code.